assets/js/base/components/cart-checkout/order-summary/index.tsx

```diff
@@ -35,7 +35,6 @@ const OrderSummary = ( {
 					{ __( 'Order summary', 'woo-gutenberg-products-block' ) }
 				</span>
 			}
-			titleTag="h2"
```

Comment:
The issue description on https://github.com/woocommerce/woocommerce-blocks/issues/8976 states the following:

> The title `Order summary` uses a `<span>` element instead of an `<h2>` element.

Given that this element will contain a few sub elements, I believe using a `<div>` is the better option here. Therefore, instead of changing `titleTag="h2"` to `titleTag="span"`, I removed that part, as `div` is [defined as the default titleTag](https://github.com/woocommerce/woocommerce-blocks/blob/28534b7c463b796f5f2a22562b87a5c2b562c6ad/packages/checkout/components/panel/index.tsx#L29).